samples/node-token-server/Index.ts

```diff
@@ -1,18 +1,18 @@
 // Copyright (c) 2005-2018 Coveo Solutions Inc.
-import { CoveoSearchTokenHandler, DynamicsPortalAuthTokenHandler, IDecodedPortalAuthTokenPayload } from "coveo-search-token-generator";
+import { CoveoSearchTokenGenerator, DynamicsPortalAuthTokenDecoder, IDecodedPortalAuthTokenPayload } from "coveo-search-token-generator";
 import * as express from "express";
 
 // -----------------------------------------------------------------------------
 // Change this configuration accordingly to represent your environment.
 const config = {
-    portalUrl: "<your_portal_url>", // example: https://yourportalurl.microsoftcrmportals.com/
-    coveoApiKey: "<your_API_key>", // The API key used to query Coveo and create a search token. It must have at least the privileges "Execute query" and "Impersonate" enabled.
+    portalUrl: "<your_portal_url>", // Example: https://yourportalurl.microsoftcrmportals.com (without slash at the end)
+    coveoApiKey: "<your_API_key>", // The API key used to query Coveo and create a search token. It must have at least the privilege "Impersonate" enabled.
     coveoPlatformUrl: "platform.cloud.coveo.com" // The URL of the Coveo Cloud V2 platform.
```

Comment:
Since "Coveo Cloud" is a product name, I would use "The Coveo Cloud URL." instead.